Guides/Releasing.md

```diff
@@ -2,9 +2,10 @@
 
 
 ## Below are some of the required changes needed before release and after adding an integration:
-#### 1. Update the path of `Source` in the pod spec to point to master branch and change version for next release version.
-#### 2. Change path of every added integration(pod) in Example projects pod file from relative path to the published path or git master branch source.
-#### 3. `git commit -am "Prepare for release X.Y.Z."` (where X.Y.Z is the new version).
-#### 4. `git tag -a X.Y.Z -m "Version X.Y.Z"` (where X.Y.Z is the new version).
-#### 5. `git push && git push --tags`.
-#### 6. `pod trunk push xxxx`, where xxxx is the newley added integration podspec.
\ No newline at end of file
+##### 1. Update the path of `Source` in the pod spec to point to master branch and change version for next release version.
+##### 2. Run ./lint.sh and .test.ch to make sure no errors detected and all tests pass.
```

Comment:
The `./lint.sh` command is properly building the project now, the previous PR was not, however the actual linter is printing a ton of errors to the terminal without specific detail. 

It looks like a hundred lines of this:
```
16 errors generated.
16 errors generated.
20 errors generated.
21 errors generated.
16 errors generated.
20 errors generated.
21 errors generated.
```

Then the actual linter outputs this with no meaningful error log for the error:
```
Linting pod spec

 -> Segment-Facebook-App-Events (1.0.4)
    - WARN  | source: Git sources should specify a tag.
    - WARN  | [iOS] license: Unable to find a license file
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the App editor. (in target 'App' from project 'App')
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | [iOS] xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 7.0, but the range of supported deployment target versions is 8.0 to 13.2.99. (in target 'Analytics' from project 'Pods')

[!] Segment-Facebook-App-Events did not pass validation, due to 1 error.
You can use the `--no-clean` option to inspect any issue.
```